### PR TITLE
fix(learn): clarify description for finders keepers

### LINKED ISSUE #38832: Finders Keepers and the "Truth Test"
**Describe your problem and how to reproduce it:**
The instructions are unclear to a new developer regarding the term "truth test". As somebody still learning I was stuck trying to figure out what a truth test was until a mentor revealed to me that it was not some sort of programming term at all but just what the second argument was being referred to as. I propose that the examples of the function that is the second argument be renamed to truthTest instead of num.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by Moonsolol**
Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38832 

Used @ojeytonwilliams wording for the fix

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers.english.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ forumTopicId: 16016
 
 ## Description
 <section id='description'>
-Create a function that looks through an array (first argument) and returns the first element in the array that passes a truth test (second argument). If no element passes the test, return undefined.
+Create a function that looks through an array, arr, and returns the first element in the array that passes a 'truth test'. That is to say, given an element x, the 'truth test' is passed if func(x) is true. If no element passes the test, return undefined.
```
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/finders-keepers.english.md:11]:
```suggestion
Create a function that looks through an array, <code>arr</code>, and returns the first element in the array that passes a 'truth test'. That is to say, given an element <code>x</code>, the 'truth test' is passed if <code>func(x)</code> is <code>true</code>. If no element passes the test, return <code>undefined</code>.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Just some formatting changes.

**Comment by kushagra010** [comment]
Hi,
I am a beginner and want to contribute. Can I work on this @Sky020 .

**Comment by Sky020** [comment]
@kushagra010 , This is a pull-request that is likely to close the issue. So, if you opened a different pull-request, this one would be prioritised.

You are welcome to work on issues with the labels _help wanted_ or _first-timers welcome_, provided there is not already an **open** PR linked to close them.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Comment by kushagra010** [comment]
Hi @thecodingaviator , can I work on this issue

**Comment by thecodingaviator** [comment]
@kushagra010 please go ahead

**Comment by Sky020** [comment]
@thecodingaviator , the issue is labelled with _first-timers only_. So, I suggest we either remove that, seeing as the issue has been up a while. Or, we will have to prioritise _first-timers_...

**Comment by thecodingaviator** [comment]
@Sky020 my bad. Last I checked, a couple of hours ago, @kushagra010 had no commits. It seems like they have had a PR merged since then. @kushagra010 if you haven't started work on the PR already, we'd like to leave the issue open for the first-timers. If you have, no worries.

**Comment by kushagra010** [comment]
@thecodingaviator I have already created a PR .

**Pull request closed without merging**